mlton/ssa/simplify.fun

```diff
@@ -284,7 +304,7 @@ fun simplify p =
                  (n + 1)
                  (List.fold
                   (!ssaPasses, p, fn ({name, doit}, p) =>
-                   maybePass ({name = name, doit = doit, midfix = midfix}, p)))
+                   maybePass ({name = name, doit = doit, execute = true, midfix = midfix}, p)))
```

Comment:
The `execute = true` should be added to the various lists that are assigned to `ssaPasses` (so that a new, in development, pass could be added with `execute = false`).